pkg/internal/test/test_doubles.go

```diff
@@ -32,6 +34,24 @@ func FromFile(filePath string) io.Reader {
 	return file
 }
 
+// NewIssueCommentEvent creates an instance of gogh.PullRequestEvent with the given values
+func NewIssueCommentEvent(payload []byte) *gogh.IssueCommentEvent {
```

Comment:
Could these two methods take a path to the file and load the content of the file in this method or is there some case when it is better to provide the bytes? If it is possible to load the file here, then I would call the method `LoadIssueCommentEvent`